config/config.r4.py

```diff
@@ -52,13 +56,16 @@
 
-coding_content='<p>This table contains p &lt; 1e-4 associations for each coding variant in FinnGen data freeze 4 (2,264 endpoints). The following gnomAD annotation categories are included: predicted loss-of-function (pLoF), low-confidence loss-of-function (LC), inframe indel, missense, start lost, stop lost. Variants have been filtered to imputation INFO score &gt; 0.6.</p><p style={{paddingBottom: \'10px\'}}>Finnish enrichment (FIN enr) is calculated as FIN AF / NFSEE AF in gnomAD 2.1, where NFSEE is non-Finnish-non-Swedish-non-Estonian European. p-values &lt; 5e-8 and Finnish enrichment &gt; 5 are in green. As the consequence and category columns are based on different genome builds (38 and 37 respectively), they differ for some variants. Hover over the column names to see their explanations, click on the column names to sort by them, and type values in the boxes below the column names to filter. Click on a variant, phenotype, or gene to get to its page.</p>'
+vis_conf = {"loglog_threshold": 10, "info_tooltip_threshold": 0.8, "manhattan_colors": ['rgb(53,0,212)', 'rgb(40, 40, 40)']}
 
-lof_content='<p>This table contains p &lt; 1e-3 associations from loss-of-function gene burden tests in FinnGen data freeze 4 (2,264 endpoints). The following VEP-annotated variants are considered: frameshift, splice donor, stop gained, splice acceptor. Association tests were run with SAIGE using the same covariates as in the regular GWAS. The dosage for each gene was calculated as the probability of carrying a minor allele in a LoF variant in the gene (1 - sum(prob(major)) where prob(major) is the genotype probability of the major allele for each variant).</p>'
+gene_pheno_export_fields=["variant.varid", "assoc.pval","assoc.beta","variant.annotation.rsids", "pheno.category", "pheno.num_cases", "pheno.num_controls", "pheno.phenocode", "pheno.phenostring", "variant.annotation.gnomad.AF_fin", "variant.annotation.gnomad.AF_nfe", "assoc.matching_results.ukbb.pval", "assoc.matching_results.ukbb.n_cases"]
 
-logo='<img src="/static/images/finngen_loop1.gif" style="float: left; width: 60px; height: 60px; margin: -10px; margin-top: 8px" />'
+about_content='content/about.html'
+coding_content='content/coding.html'
+lof_content='content/lof.html'
```

Comment:
Needs to change to full path or we need to ensure that the location where the server is run contains this data